src/app/containers/RadioPageBlocks/Blocks/AudioPlayer/StructuredData.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { string, number } from 'prop-types';
+
+const StructuredData = ({
+  promoBrandTitle,
+  shortSynopsis,
+  durationISO8601,
+  embedUrl,
+  imageUrl,
+  episodeAvailableFrom,
+}) => {
+  const metadata = {
+    '@context': 'http://schema.org',
+    '@type': 'AudioObject',
+    name: promoBrandTitle,
+    description: shortSynopsis,
+    duration: durationISO8601,
+    thumbnailUrl: imageUrl,
+    uploadDate: new Date(episodeAvailableFrom).toISOString(),
+    embedURL: embedUrl,
+  };
+
+  return (
+    <Helmet>
+      <script type="application/ld+json">{JSON.stringify(metadata)}</script>
```

Comment:
You might need to serialise the JSON just in case: see https://github.com/bbc/simorgh/blob/latest/src/app/containers/LinkedData/index.jsx#L91 as an example